Formula/tarantool.rb

```diff
@@ -1,10 +1,8 @@
 class Tarantool < Formula
   desc "In-memory database and Lua application server."
   homepage "https://tarantool.org/"
-  url "http://download.tarantool.org/tarantool/1.6/src/tarantool-1.6.8.772.tar.gz"
-  version "1.6.8-772"
-  sha256 "e07913d3416fcf855071e7b82eed0c5bcdb81a6e587fa2d900a9755ed5bb220c"
-  revision 2
+  url "https://download.tarantool.org/tarantool/1.7/src/tarantool-1.7.3.7.tar.gz"
```

Comment:
You may want to create a release with the GitHub GUI (right now it implies 1.7.3 beta is latest), and upload this tarball so we can add it here as a mirror too, but we can always add that later if the spirit moves you.